src/app/components/patient-tests-table/patient-tests-table.component.less

```diff
@@ -223,7 +222,7 @@
   margin-right: 15px;
 }
 .count-checked-tests span {
-  color: #3f51b5;
+  color:#07676c;
```

Comment:
Why you don't use variable from colors.less here?

Reply:
Done